scripts/test-miri.sh

```diff
@@ -5,4 +5,8 @@ here="$(dirname "$0")"
 src_root="$(readlink -f "${here}/..")"
 cd "${src_root}"
 # miri is very slow; so only run very few of selective tests!
-./cargo nightly miri test -p solana-hash -p solana-account-info
+#
+# Note: fixed version of solana-hash is needed here to avoid
+#       dependency resolution issues since v3 just re-exports
+#       v4 of the `solana-hash` crate.
+./cargo nightly miri test -p solana-hash@3.0.0 -p solana-account-info
```

Comment:
This was needed since miri was complaining that there were 2 `solana-hash` crates available.

Reply:
I think we can actually remove `solana-hash` from this script, since there are no tests in the crate.